### DISCUSSION TITLE
[Legends Component] Customization for Legends overflow

### DISCUSSION THREAD

**Discussion by AllenLin8** [microsoft/fluentui#32855] [Q&A]
Hello!

I see that [there is a lot of flexibility](https://github.com/microsoft/fluentui/blob/603fe594debaf719b6b1804da405e9ffecf0a220/packages/react-charting/src/components/Legends/Legends.types.ts#L192) within the Legends component for custom overflow. 

Is there any way to modify these props (or any other attributes of the Legends that may be applicable) to make a Legend appear with two lines rather than just one, but still with the standard overflow text and overflow button?

![image](https://github.com/user-attachments/assets/2d7c8a22-eeac-4871-ab61-4119389285a1) https://developer.microsoft.com/en-us/fluentui#/controls/web/legends

^ instead of just the one line across above with the 10 legends, have it appear on two lines **with** the "7 Overflow Items" button.

With this question above, any general advice/examples on how to utilize the Legends overflow customization options would also be appreciated. Thank you!

**Comment by AtishayMsft** [comment]
Hey, we don't support this customization for now. You could either have overflow or wrapping of lines for the legends.
Though we are coming up with the functionality of pagination of legends if the vertical height is limited to handle all the wrapped legend items.

**Reply by AllenLin8** [reply]
Thank you for the reply! Do you happen to know what the timeline is for the legend pagination (ex. rough estimate of when that change will be introduced)?

**Comment by AtishayMsft** [comment]
This will come in the next 3-4 months.
